src/main/java/lotto/domain/LottoManager.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public enum LottoManager {
+    INSTANCE;
+
+    private static final int LOTTO_PRICE = 1000;
+
+    public Integer getPurchasableLottos(Long purchaseAmount) {
+        return (int) (purchaseAmount / LOTTO_PRICE);
+    }
+
+    public List<Lotto> purchaseLotto(int count) {
+        List<Lotto> lottos = new ArrayList<>();
+        for (int i = 0; i < count; i++) {
+            List<Integer> numbers = Lotto.createNumbers();
+            Lotto lotto = new Lotto(numbers);
```

Comment:
```suggestion
            Lotto lotto = new Lotto(Lotto.createNumbers());
```

메서드에서 어떤게 들어가는지 알 수 있는것 같아서 합쳐도 될 것 같아요!

Reply:
합친 것을 보니 불필요한 임시 변수를 제가 사용한 것 같네요!